### PR TITLE
Adding Testing Checklist to UX Guidelines

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/developer/fx-guidelines.rst [hunk 1]
```diff
@@ -459,6 +459,38 @@ All inputs such as textareas should behave consistently, if the default behaviou
         :alt: Edit header modal
         :width: 500 px
 
+Testing Checklist 
+************************
+
+In addition to above guidelines, the below provides a concrete checklist of mistakes to watch for whem reviewing proposed product changes.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: `whem` > `when` ?

File: source/developer/fx-guidelines.rst [hunk 2]
```diff
@@ -459,6 +459,38 @@ All inputs such as textareas should behave consistently, if the default behaviou
...
+
+..  image:: https://cloud.githubusercontent.com/assets/177788/17340912/1599a0aa-58a7-11e6-94e3-1e2a0895c40f.png
+
+**2\) Is the WHITESPACE next to icons EVEN?** 
+
+Example of uneven icon spacing: 
+
+..  image:: https://cloud.githubusercontent.com/assets/177788/17340912/1599a0aa-58a7-11e6-94e3-1e2a0895c40f.png
+
+**3\) Are there visual GAPS**
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: `Are there visual GAPS` > `Are there visual GAPS?` ?

**Comment by it33** [comment]
Thanks @lindy65!

**Comment by lindy65** [comment]
+1

File: source/developer/fx-guidelines.rst [hunk 3]
```diff
@@ -459,6 +459,38 @@ All inputs such as textareas should behave consistently, if the default behaviou
...
 
+Testing Checklist 
+************************
+
+In addition to above guidelines, the below provides a concrete checklist of mistakes to watch for when reviewing proposed product changes. 
+
+User Experience Checklist 
+------------------------------
+
+**1\) Is the WHITESPACE next to icons SUFFICIENTLY SPACED?**
```
Comment (jasonblais) [inline source/developer/fx-guidelines.rst]: we should probably get consistent with list labelling: `1\)` used here, `1 -` used in SAML docs.

**Comment by jasonblais** [comment]
+1 other than comment above, note to doc owner

**Pull request merged**